### ISSUE TITLE
ansible-lint incorrectly flags shell script called "service" as needing to use service module

### ISSUE THREAD

**Issue by dawn-minion** [ansible/ansible-lint#367]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible 2.6.3
  config file = /home/dm9999/.ansible/ansible.cfg
  configured module search path = [u'/home/dm9999/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/dm9999/.local/lib/python2.7/site-packages/ansible
  executable location = /home/dm9999/.local/bin/ansible
  python version = 2.7.15rc1 (default, Apr 15 2018, 21:51:34) [GCC 7.3.0]


ansible-lint 3.4.23
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

We have a shell script called `service` - It is unrelated to the system `service` command or module. When we call the script, the linter should not flag it, saying we need to use the `service` module.

# Actual Behaviour (Bug report only)

Example playbook with issue:
```
- name: Do the things
  tasks:
    - name: Do some things
      command: |
        /path/to/service doThingA
```

Ouput from ansible-lint:
```
[ANSIBLE0006] service used in place of service module
service.yml:5
Task/Handler: Do some things
```

**Comment by dawn-minion** [comment]
@webknjaz Is there any reason why this was closed as wont fix? It's quite annoying having to exclude every single usage of our script when the rule is incorrect.

**Comment by webknjaz** [comment]
The rule is correct. The proper fix for you would be to wrap it as an Ansible module.

**Comment by dawn-minion** [comment]
@webknjaz How is it correct, though? The rule is if you're using the system service command, it's recommended to use the Ansible system module instead. However, our command is completely unrelated to the Sys V command - different path, different purpose. The rule is flagging _any_ command called "service", instead of _only_ the one for Sys V scripts.

**Comment by webknjaz** [comment]
Yes, but also it's not recommended to use shell commands and instead, you should wrap them into modules. Also, there's no robust way to check if you're using a system service management binary or something else. It's a linter and it shouldn't run any heavy code.
Besides, you can just disable it globally via  `-x 303`.
